src/responder/multiResponder.ts

```diff
@@ -207,4 +211,14 @@ export class MultiResponder {
             logger.error(doh);
         }
     }
+    /**
+     * Checks to see if the responder balance is lower than the threshold set in the constructor.
+     * If the balance is lower, a warning will be outputted by the logger
+     */
+    public async checkBalance(){
+        const currentBalance = await this.signer.provider!.getBalance(this.address);
+        if(currentBalance.gt(this.balanceThreshold)){
```

Comment:
Woops, I've just realised that this check is the wrong way round. Should be `.lt`

Reply:
Thanks for pointing this out, I switched it to test something and forgot to change it back. There would have been a lot of low balance warnings issued lol